content/writing_tests/tutorial.md

```diff
@@ -0,0 +1,209 @@
+---
+title : "Tutorial"
+date :  2019-09-12T13:16:37+02:00
+weight : 7
+---
+
+# Tutorial
+
+After learning about the native and DOM-based testing capabilities of Sakuli, show a demonstration what you probably need
+to set up a monitoring check. The 10 do's and don'ts is a nice guideline to start, on what should be checked.
+
+We will use the sakuli docs page, which you are on right now, but starting from the sakuli.io page.
+The complete test will be at the bottom of the page.
+
+## Setting up a Sakuli Test Suite
+
+Sakuli will bootstrap a test suite with following command. A detailed explanation on what files are needed, can be found
+[in anatomy chapter](anatomy.md).
+{{< highlight bash >}}
+npx @sakuli/cli create project . --package
+npm i
+{{< /highlight >}}
+
+### Typescript
+
+This step is optional but we recommend using typescript. For example, in case of a missing `await` typescript will throw
+an error during compilation. But this tutorial will also work if you decide to not use typescript.
+ 
+{{< highlight bash >}}
+npx sakuli enable-typescript .
+{{< /highlight >}}
+
+We also need to rename `sakuli_test_suite/case1/check.js` to `check.ts`
+
+## Native-testing Tutorial
+
+The example for this tutorial is to open the `10 Step Guide to E2E Monitoring` whitepaper only with native interaction
+and download the . For this we are going to set `testsuite.uiOnly=true` in the testsuite.properties.
+
+First we are going to open a chromium browser and navigate to the whitepaper section to download the whitepaper.
+{{< highlight typescript >}}
+(async () => {
+  const tc = new TestCase();
+  const env = new Environment();
+  const chromium = new Application('chromium-browser --incognito');
+  const screen = new Region();
+
+  await tc.addImagePaths('./assets');
+  try {
+    await env.setSimilarity(0.99);
+
+    await chromium.setSleepTime(3);
+    await chromium.open();
+
+    await env.type('sakuli.io/wp')
+      .type(Key.ENTER)
+      .getRegionFromFocusedWindow()
+      .waitForImage('open_whitepaper.png', 3)
+      .mouseMove()
+      .click();
+    tc.endOfStep('open whitepaper');
+
+  } catch (e) {
+    await tc.handleException(e);
+  } finally {
+    tc.saveResult();
+  }
+})();
+{{< /highlight >}}
+
+So let's have a closer look on the code. First we are want to open the chromium in incognito mode so we added the parameter.
```

Comment:
```suggestion
So let's have a closer look to the code. As a first step we want to open the chromium browser and use incognito mode. This can be achieved by opening the application with the respective parameter:
`const chromium = new Application('chromium-browser --incognito');`
```